app.js

```diff
@@ -8,6 +8,15 @@ var app = express()
 var bodyParser = require('body-parser')
 var mongo = require('express-mongo-db')
 
+// Load configs
+try {
+    var configs = require('./conf/config.js')
+} catch (ex) {
+    console.error('Could not load configs. Did you forget to create them?')
```

Comment:
Where do these (`console.error` and `console.info`) get displayed? The terminal or the browser console?

Reply:
The terminal stdout. So, for us, it would be the log file.